src/apps/content-editor/src/app/views/ItemEdit/components/Header/ItemVersioning/ScheduleFlyout/ScheduleFlyout.js

```diff
@@ -35,12 +35,7 @@ export default class ScheduleFlyout extends Component {
       userTimezone: userTimezone,
       selectedTimezone: userTimezone,
       selectedTime: moment().format(UTC_FORMAT),
-      timezones: moment.tz.names().map((el, i) => {
-        return {
-          value: el,
-          html: el,
-        };
-      }),
+      timezones: moment.tz.names(),
```

Comment:
How is it that the array returned by `moment.tz.names()` fits the data shape that the `Autocomplete` component needs?